Sources/Elementary/Core/Html+Attributes.swift

```diff
@@ -117,6 +126,15 @@ public extension HTML where Tag: HTMLTrait.Attributes.Global {
     func attributes(_ attributes: HTMLAttribute<Tag>..., when condition: Bool = true) -> _AttributedElement<Self> {
         _AttributedElement(content: self, attributes: .init(condition ? attributes : []))
     }
+
+    /// Adds the specified attributes to the element.
+    /// - Parameters:
+    ///   - attributes: The attributes to add to the element as an array.
+    ///   - condition: If set to false, the attributes will not be added.
+    /// - Returns: A new element with the specified attributes added.
+    func attributes(_ attributes: [HTMLAttribute<Tag>], when condition: Bool = true) -> _AttributedElement<Self> {
+        _AttributedElement(content: self, attributes: .init(condition ? attributes : []))
+    }
```

Comment:
also here, I was thinking a label might be better... maybe `attributes(elemenetsOf: myArray)`?

Reply:
what about `attributes(contentsOf: myArray)`, this would mirror other collections' methods?

Reply:
oh, yes, that was what I was going for (just remembered it wrong) ; )